.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,81 @@
+name: Bug report
+description: Report incorrect behavior in the Modin library
+title: 'BUG: '
+labels: ['bug 🦗', 'Triage 🩹']
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Modin version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the latest version of Modin.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the main branch of Modin.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a minimal, copy-pastable example.
+      placeholder: >
+        import modin.pandas as pd
+
+        df = pd.DataFrame(range(5))
+
+        ...
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Issue Description
+      description: >
+        Please provide a description of the issue shown in the reproducible example.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: >
+        Please describe or show a code example of the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Error Logs
+      description: >
+        Please paste the output of any relevant error logs.
+      value: >
```

Comment:
this should probably include a placeholder stating how to obtain modin logs

Reply:
Do you mean like the Modin logging module? We usually haven't asked those from users. Error logs/stack trace should be all that we need to debug. I'd rather not further complicate this by asking for every single thing.

Reply:
+1 to @pyrito ; I doubt many people would even put the modin logs if we asked for them. In my experience having a reproducible example and a stack trace are much more useful than modin logs. Modin logs are more useful for performance issues, which people don't seem to post on github much anyway.